cmd/google_cloud_ops_agent_engine/main.go

```diff
@@ -59,6 +60,11 @@ func run() error {
 		return err
 	}
 
+	err = self_metrics.CollectOpsAgentSelfMetricsToOTLPJSON(ctx, uc, uc)
```

Comment:
Let's also also add this to the UAP Plugin service.

Reply:
Where would this go in the UAP plugin service ?

Reply:
Looking more closely, this is already taken care of, since we are also running confgenerator on the Plugin.